src/io_proto.c

```diff
@@ -499,12 +499,13 @@ static int32_t io_encode_open_request(
 )
 {
     int32_t result;
-    __io_encode_type_begin(ctx, request, 2);
+    __io_encode_type_begin(ctx, request, 3);
```

Comment:
Yeah, this is the msgpack properties count.  Would probably make sense to document this as part of the __io... macro.